builtin/mainmenu/tab_online.lua

```diff
@@ -312,7 +312,7 @@ local function main_button_handler(tabview, fields, name, tabdata)
 		return true
 	end
 
-	if (fields.Dte_search or fields.btn_mp_search) and not
+	if fields.Dte_search and not
```

Comment:
this broke search on Android

Reply:
I don't see why this would break search (since if `fields.Dte_search` is nil then it would crash anyway). I've pushed a different fix, though I still don't know why `fields.Dte_search` is nil in the first place.

Reply:
How do you search if the search button doesn't work?